src/mpi/romio/configure.ac

```diff
@@ -777,6 +777,46 @@ if test -n "$FILE_SYSTEM" ; then
    done
 fi
 
+AC_ARG_WITH([cart],
+    [AS_HELP_STRING([--with-cart],
+            [Build DAOS ROMIO driver[default=no]])],,
```

Comment:
I don't think this help string is what you mean it to be?    You are trying to tell ROMIO where to find the CART libraries, right?   I grew up with using `--with-whatever` this way, too, but it's more autoconf-y to rely on LIBS, CPPFLAGS, etc environment variables.    Does CART provide a pkg-config file?  If not, this approach is fine for now, but can you open an issue on the CART page?

Reply:
they don't. 
we have talked about both cart and daos provide pkg-config files and it's in our TODO list, so will update this once we have something. (probably towards 1.0)